code.js

```diff
@@ -1,3 +1,39 @@
-function divideAndConquerSum(a) {
-    return a;
+function divideAndConquerSum(array) {
+    let sum = 0;
+    switch(array.length){
+        case 0:
+            return 0;
+            break
+        case 1:
+            return array[0];
+            break;
+        case 2:
+            return array[0] + array[1];
+            break;
+        default:
+            sum = helper(array);
+            return sum;
+    }  
+}
+
+function helper(arr){
+    switch(arr.length){
+        case 1:
```

Comment:
Same question as before. There's a lot of duplicated code in this and the previous function.

Reply:
I have put it all in one function and that made it a lot more simple now.